specificatie/verblijfplaats.yaml

```diff
@@ -228,10 +210,6 @@ components:
             $ref: '#/components/schemas/Adresregel2'
           adresregel3:
             $ref: '#/components/schemas/Adresregel3'
```

Comment:
zouden we in verblijfplaats de adresregels niet anders noemen dan in adressering?

Reply:
Nu je het zegt kan ik me vaag herinneren dat we het daar over gehad hebben. Heb ik helaas geen aantekening van gemaakt. Ik heb er nu adresseringregel1 etc van gemaakt, maar weet niet of dat nu een goede naam is.

Reply:
ik denk dat adresseringregel1 verwarrend kan zijn, want adressering bestaat uit maximaal 5 regels:
- regel 1: aanspreekvorm
- regel 2: aanschrijfwijze
- regel 3: adresregel 1 (straat + huisnummer)
- regel 4: adresregel 2 (postcode + woonplaats)
- regel 4: adresregel 3 (regel 3 adres buitenland)
- regel 5: land

Onze 3 adresregels gaan alleen over de drie regels die het adres bevatten, vandaar dat "adresregel" best een mooie naam is.

Waarom niet de buitenlandse adresregels aanpassen naar regel1AdresBuitenland, enz.? Dan komt het helemaal overeen met de naam in LO GBA.

Reply:
Ik heb de adresregels in adres Buitenland nu regel1, regel2 en regel3 genoemd. de toevoeging AdresBuitenland is dubel omdat deze al in de groep verblijfplaatsBuitenland zitten.